spacy/vocab.pyx

```diff
@@ -164,7 +164,7 @@ cdef class Vocab:
         if len(string) < 3 or self.length < 10000:
             mem = self.mem
         cdef bint is_oov = mem is not self.mem
-        lex = <LexemeC*>mem.alloc(sizeof(LexemeC), 1)
+        lex = <LexemeC*>mem.alloc(1, sizeof(LexemeC))
```

Comment:
I'm a little baffled by this one, because this has been in the code for years. But the fix is right, yes? no? I'm so confused. I guess it just worked magically?

Reply:
I can never remember which way calloc's arguments work =/. It should be equivalent regardless, right? The code in `Pool.alloc` seems to only use the two multiplied together.

Reply:
Ah! Yes ofcourse. That explains why it was never an issue :-)